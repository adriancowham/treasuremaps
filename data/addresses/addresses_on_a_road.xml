<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339786591.html"> 
<title><![CDATA[HUGE BARN SALE (PLYMOUTH)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339786591.html</link> 
<description><![CDATA[18240 Latrobe rd. Plymouth Ca.
<br> 

<br> 
HUGE BARN SALE! ESTATE SALE! THIS FRIDAY, SATURDAY &  SUNDAY!, yes this easter sunday!
<br> 

<br> 
8am to 5pm ALL THREE DAYS
<br> 

<br> 
Everything Must Go!
<br> 

<br> 
Antiques
<br> 
Carnival ware
<br> 
Tube Radios
<br> 
Vintage closets
<br> 
Collectables, lots of princess Di, Elvis 
<br> 
furniture, appliances, kitch table, knick knack, iron pots, iron stove, FISHING GEAR! poles, lures, etc
<br> 

<br> 
Toys, clothes you name it its there!
<br> 

<br> 

<br> 

<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=PLYMOUTH -->Location: PLYMOUTH
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T18:56:51-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339786591.html</dc:source> 
<dc:title><![CDATA[HUGE BARN SALE (PLYMOUTH)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T18:56:51-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339978522.html"> 
<title><![CDATA[Lots of Quality Stuff (Auburn)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339978522.html</link> 
<description><![CDATA[Lots of Quality Stuff
<br> 

<br> 
3675 Bell Rd Auburn CA 95603 Friday and Saturday April 22/23 8am to 5pm
<br> 

<br> 
DVDs $2 ea –Lots of VHS $0.50 each
<br> 

<br> 
Old Tools, Electric Hedge Trimmers, Aluminum Ladder, Hose Reels,
<br> 

<br> 
Doll collection, TY Beanie Babies   
<br> 

<br> 
Interior and exterior doors, Small and Xsmall scrubs, St. Josephs’ School Uniforms
<br> 

<br> 
Antiques, Chevy parts (’72 truck?)
<br> 

<br> 

<br> 
<a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF0030.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF0030.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF0021.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF0021.jpg" border="0"></a>
<br> 
<a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9908.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9908.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9844-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9844-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9731-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9731-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9732-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9732-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9736-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9736-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9783-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9783-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9716.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9716.jpg" border="0"></a>
<br> 
<a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9642-1.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9642-1.jpg" border="0"></a> <a href="http://s955.photobucket.com/albums/ae39/bignickle/?action=view&amp;current=DSCF9593.jpg" target="_blank" rel="nofollow"><img src="http://i955.photobucket.com/albums/ae39/bignickle/DSCF9593.jpg" border="0"></a>
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Auburn -->Location: Auburn
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:55:15-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339978522.html</dc:source> 
<dc:title><![CDATA[Lots of Quality Stuff (Auburn)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:55:15-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339772536.html"> 
<title><![CDATA[Huge Estate Liquidation Sale this Friday-Saturday 10am-4pm (Sacramento)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339772536.html</link> 
<description><![CDATA[  Huge Estate Liquidation Sale this Friday-Saturday from 10am-4pm. Everything is priced to move. Furniture, Appliances, Antiques, Collectibles, Art, Electronics, Video Games Systems, Speakers, Kitchen Items, Toys, Bikes, Tools, Car Stuff, Boat Motors, Motorcycle Stuff, Books, CD's, DVD's, Records, VHS, Cassettes, Fishing Gear, Camping Gear, Sporting Goods, Barbie Collection New In Box's, Hotwheel Collection, Crystal, Princess House, and so much more. 7000 square feet of savings. Estate liquidators bringing the savings directly to you. Located at 10275 Old Placerville Rd. Suite #4 Sacramento 95827. Near Hwy 50 & Bradshaw. 
<br> 

<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Sacramento -->Location: Sacramento
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T18:40:17-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339772536.html</dc:source> 
<dc:title><![CDATA[Huge Estate Liquidation Sale this Friday-Saturday 10am-4pm (Sacramento)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T18:40:17-07:00</dcterms:issued> 
</item> 

</posts>