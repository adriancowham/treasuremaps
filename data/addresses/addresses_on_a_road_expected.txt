18240 Latrobe rd.
3675 Bell Rd
10275 Old Placerville Rd.